packages/api/pages/api/languages/[code]/members/index.ts

```diff
@@ -0,0 +1,146 @@
+import * as z from 'zod';
+import {
+  GetLanguageMembersResponseBody,
+  LanguageRole,
+  PostLanguageMemberRequestBody,
+} from '@translation/api-types';
+import { client, PrismaTypes } from '../../../../../shared/db';
+import createRoute from '../../../../../shared/Route';
+import { authorize } from '../../../../../shared/access-control/authorize';
+import { NotFoundError } from '../../../../../shared/errors';
+import { auth } from '../../../../../shared/auth';
+import mailer from '../../../../../shared/mailer';
+import { randomBytes } from 'crypto';
+import { origin } from '../../../../../shared/env';
+
+export default createRoute<{ code: string }>()
+  .get<void, GetLanguageMembersResponseBody>({
+    authorize: authorize((req) => ({
+      action: 'read',
+      subject: 'Language',
+      subjectId: req.query.code,
+    })),
+    async handler(req, res) {
+      const language = await client.language.findUnique({
+        where: {
+          code: req.query.code,
+        },
+      });
+
+      if (language) {
+        const users = await client.authUser.findMany({
+          where: {
+            languageRoles: {
+              some: {
+                languageId: language.id,
+              },
+            },
+          },
+          include: {
+            languageRoles: {
+              where: {
+                languageId: language.id,
+              },
+            },
+          },
+        });
+
+        res.ok({
+          data: users.map((user) => ({
+            userId: user.id,
+            name: user.name ?? undefined,
+            email: user.email ?? undefined,
+            roles: user.languageRoles
+              .map((role) => role.role)
+              .filter(
+                (role): role is LanguageRole =>
+                  role !== PrismaTypes.LanguageRole.VIEWER
+              ),
+          })),
+        });
+      } else {
+        res.notFound();
+      }
+    },
+  })
+  .post<PostLanguageMemberRequestBody, void>({
+    authorize: authorize((req) => ({
+      action: 'administer',
+      subject: 'Language',
+      subjectId: req.query.code,
+    })),
+    schema: z.object({
+      email: z.string(),
+      roles: z.array(
+        z.enum(Object.values(LanguageRole) as [LanguageRole, ...LanguageRole[]])
```

Comment:
What is the difference between this type definition and 

`z.enum(Object.values(LanguageRole) as LanguageRole[])`